locale/gl/LC_MESSAGES/django.po

```diff
@@ -2785,7 +2785,7 @@ msgstr ""
 #: templates/zerver/social_auth_select_email.html:7
 msgid ""
 "\n"
-"        <h1>Select email</h1>\n"
```

Comment:
Can you discard this commit? These changes aren't necessary as @timabbott is responsible for pulling data from Transifex and updating the files in `locale` periodically.